CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.7.1
```

Comment:
```suggestion
## 0.8.0
```

Reply:
Can you also update https://github.com/XPBytes/media_types-serialization/blob/master/lib/media_types/serialization/version.rb#L3 ?